force-app/main/default/classes/SOQL.cls

```diff
@@ -233,11 +272,14 @@ public with sharing class SOQL {
     public static Map<Id, String> getAccountIdsToNamesNotInCountry(String country) {
         // Create a map to hold the Account Ids to Account Names
         Map<Id, String> accountIdsToNames = new Map<Id, String>();
-
         // Query the Accounts
+        List<Account> myAccounts = [SELECT Id, Name FROM Account WHERE BillingCountry != :country];
         // Loop through each Account and add the Id and Name to the map
+        for (Account acc : myAccounts) {
+            accountIdsToNames.put(acc.Id, acc.Name);
+        }
```

Comment:
You can do it like this. There is a shortcut to put the query right into the map. Try it that way.